md2cf/api.py

```diff
@@ -66,7 +66,11 @@ def __init__(
     def _request(self, method, path, **kwargs):
         r = self.api.request(method, urljoin(self.host, path), **kwargs)
         r.raise_for_status()
-        return bunchify(r.json())
+        
+        if r.status_code != 204:
+            return bunchify(r.json())
+        else:
+            return None
```

Comment:
What's the purpose of this block?

Reply:
For some reason, the Confluence API returns a 204, rather then a 200 when deleting pages (seems to be documented here https://jira.atlassian.com/browse/CONFCLOUD-70605). Bunchify failed on this response, so I skipped it. Probably a different part of the response should be used here...